public/stations/index.html

```diff
@@ -2,96 +2,89 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Chargetrip API: station map example</title>
+    <title>Chargetrip API: station example</title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <meta
-      name="description"
-      content="Show charging stations on the map with the Chargetrip API. Filter stations based on a power or amenities available nearby."
-    />
-    <meta
-      name="keywords"
-      content="Chargetrip, Chargetrip API, EV, ev-cars, ev-routing, tile service, clusterbuster, map, filters"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;600&display=swap"
-      rel="stylesheet"
-    />
+    <meta name="description" content="Query particular station with the Chargetrip API." />
+    <meta name="keywords" content="Chargetrip, Chargetrip API, EV, ev-cars, ev-routing, station" />
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet" />
     <link rel="icon" href="../favicon-57.png" type="image/x-icon" />
     <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.1/mapbox-gl.css" rel="stylesheet" />
     <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
     <div id="map"></div>
-    <div class="github-link">
-      <a href="https://github.com/chargetrip/examples/tree/master/stations" title="github project">
-        View on Github
-      </a>
-      <button aria-label="remove github link" title="remove github link" onclick="this.parentElement.remove()"></button>
-    </div>
-
-    <button id="centerMe" class="centerMe-button">📌</button>
-
-    <div class="station-filters">
+    <div class="card">
       <header>
-        <div>Showing <span id="stationAmount"></span> stations</div>
-        <div class="subtext">Note: example is limited to 20 stations</div>
+        <h1>13 stations</h1>
```

Comment:
The 13 stations should be a variable and update depending on how many stations are shown.